systemtests/bin/bconsole

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 # wrapper script to start/stop bconsole for each individual test
 . ./environment
-${bin}/bconsole -c ${conf}
+${bin}/bconsole -c ${conf} $*
```

Comment:
Using $* will apply word-splitting to your parameters, this is usually not what you want.
```suggestion
${bin}/bconsole -c ${conf} "$@"
```